### PR TITLE
Load polyfills only if browser is IE

### PR DESCRIPTION
**Pull request by theasteve**

### CODE REVIEW

File: wp-content/themes/access/src/js/main.js [hunk 1]
```diff
@@ -1,29 +1,6 @@
...
-
-// ACCESS Patterns Polyfills
-import 'utilities/element/matches';
-import 'utilities/element/closest';
-import 'utilities/element/remove';
-import 'utilities/nodelist/foreach';
-
 // Core Modules
-// import Utility from 'modules/utility';
+import Utility from 'modules/utility';
```
Comment (devowhippit) [inline wp-content/themes/access/src/js/main.js:3]: This module shouldn't be loaded into main.js

**Review by devowhippit — CHANGES_REQUESTED** [review]
@theasteve looks like a good approach. Does this account for the polyfills in the other scripts? "archive-location.js", "archive-programs.js", "screener.js", single-location.js", "single-programs.js" ?

**Comment by theasteve** [comment]
> @theasteve looks like a good approach. Does this account for the polyfills in the other scripts? "archive-location.js", "archive-programs.js", "screener.js", single-location.js", "single-programs.js" ?

@devowhippit it seems to work for most of the polyfills in those files, althought I notice an error in IE 
Screen Shot 2020-10-26 at 1 16 26 PM coming from the screener.js file. I wonder if this is an issue coming from this implementation.

**Review by devowhippit — CHANGES_REQUESTED** [review]
@theasteve You need to make sure that all of the other scripts polyfills are included in the new file you are creating. A bug in IE indicates that this is not ready for review. Please included the polyfills that exist in all of the other files I mentioned and make sure that IE is clear of errors in the console before assigning back to me. DM with any questions regarding this.

**Comment by theasteve** [comment]
> @theasteve You need to make sure that all of the other scripts polyfills are included in the new file you are creating. A bug in IE indicates that this is not ready for review. Please include the polyfills that exist in all of the other files I mentioned in the new polyfill script you are creating and make sure that IE is clear of errors in the console before assigning back to me. DM with any questions regarding this.

Screen Shot 2020-10-27 at 4 55 01 PM
Screen Shot 2020-10-27 at 5 00 39 PM
No errors are being displayed in new implemetation of Polyfills

**Review by devowhippit — CHANGES_REQUESTED** [review]
Getting close! **main.js** is not the only script that uses polyfills. You've enqueued the script there but their module deps will need to be moved into the new script.

* **A)** Remove polyfills from other view scripts and **B)** move them into the main **polyfill.js**. Do **(B)** only _if_ they are not imported into **polyfill.js** already.
* Make the IE conditional DRY. Add function `is_ie() { ...` to **lib/functions.php** and call that method when enqueueing the script `if (is_ie()) { ...`

DM me with any Qs.

**Comment by theasteve** [comment]
> Getting close! **main.js** is not the only script that uses polyfills. You've enqueued the script there but their module deps will need to be moved into the new script.
> 
>     * **A)** Remove polyfills from other view scripts and **B)** move them into the main **polyfill.js**. Do **(B)** only _if_ they are not imported into **polyfill.js** already.
> 
>     * Make the IE conditional DRY. Add function `is_ie() { ...` to **lib/functions.php** and call that method when enqueueing the script `if (is_ie()) { ...`
> 
> 
> DM me with any Qs.

@devowhippit I tested the pages on IE, no errors are being logged into the console. I check the `polyfill.js` for duplicates

**Review by devowhippit — APPROVED** [review]
@theasteve Awesome, great work!

**Pull request merged**